src/components/shared/footer.tsx

```diff
@@ -12,8 +13,11 @@ export const Footer: React.FunctionComponent = () => (
     <GovtechImage className="mb-4" src={govtechLogo} alt="govtech logo" />
     <p className="small">©2020 Government of Singapore | Last updated 23rd May 2020</p>
     <p className="small">
+      <Link to="/terms" className="text-primary mx-3">
```

Comment:
when clicking on the link navigation stays at the bottom, we should go to the top